language/xsp/model/input_pipeline.py

```diff
@@ -24,7 +24,7 @@
 from language.xsp.model import sequence_example_decoder
 
 from six.moves import zip
-import tensorflow.compat.v1 as tf
```

Comment:
Could you leave these tensorfow.compat.v1 calls in for internal correctness?

Reply:
I use tensorflow 1.15.0, which breaks when trying to do `tf.contrib` if `tf` is imported this way (the error is specifically `AttributeError: module 'tensorflow._api.v1.compat.v1' has no attribute 'contrib'`). Should I be using a different version of tensorflow?

Reply:
Ah I didn't realize contrib wasn't accessible that way. What you have here is good for now. I'll follow up with additional changes to ensure internal compatibility as well.

Reply:
If you'd like, I can also make this consistent within my part of the repo (i.e., `xsp`), because I only changed the lines that broke the import.

Reply:
It would be great if you could do this on your end instead. The plan was to
(1) Minimize or remove tf.contrib usage. I believe 
  - tf.contrib.tpu.* can be replaced with tf.estimator.tpu.*
  - tf.contrib.slim can be replaced with https://github.com/google-research/tf-slim
  - tf.contrib.training.checkpoints_iterator can be replace with tf.train.checkpoints_iterator

(2) If any of those don't work, you can import the original contrib packages with
  from tensorflow.contrib import slim as contrib_slim

(3) Reinstate the all the `import tensorflow.compat.v1 as tf` calls.

Reply:
In fact, if you can remove all of the contrib dependencies, the code should work with the latest version of TF.

Reply:
Great, thanks, I'll try these out!